### PR TITLE
fix(wasm): fixed time.Sleep and Sign issues on wasm

### PR DESCRIPTION
**Pull request by cnlangzi**
A brief description of the changes in this PR:

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zboxcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- gosdk: https://github.com/0chain/gosdk/pull/413
- zboxcli: https://github.com/0chain/zboxcli/pull/190
- js-client-sdk :   https://github.com/0chain/js-client-sdk/pull/103

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | fix/time_sleep_wasm  |
| zboxcli  | fix/time_sleep_wasm  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: cmd/walletinfo.go [hunk 1]
```diff
@@ -74,7 +75,7 @@ var signCmd = &cobra.Command{
 		} else {
 			data = encryption.Hash(data)
 		}
-		sign, err := client.Sign(data)
+		sign, err := sys.Sign(data, client.GetClient().SignatureScheme, client.GetClientSysKeys())
```
Comment (peterlimg) [inline cmd/walletinfo.go]: We don't need this change if the gosdk defined the client.Sign function as I suggested

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | fix/time_sleep_wasm  |
| zboxcli  | fix/time_sleep_wasm  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**